examples/with-vuetify/nuxt.config.js

```diff
@@ -1,31 +1,63 @@
-import VuetifyLoaderPlugin from 'vuetify-loader/lib/plugin'
-
 export default {
+  mode: 'universal',
   /*
-  ** Head elements
-  ** Add Roboto font and Material Icons
-  */
+   ** Headers of the page
+   */
   head: {
-    link: [
+    titleTemplate: '%s - ' + process.env.npm_package_name,
+    title: process.env.npm_package_name || '',
+    meta: [
+      { charset: 'utf-8' },
+      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
       {
-        rel: 'stylesheet',
-        type: 'text/css',
-        href: 'https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons'
+        hid: 'description',
+        name: 'description',
+        content: process.env.npm_package_description || ''
       }
-    ]
-  },
-
-  build: {
-    plugins: [new VuetifyLoaderPlugin()],
-    extractCSS: true,
-    transpile: ['vuetify/lib']
+    ],
+    link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }]
   },
   /*
-  ** Load Vuetify into the app
-  */
-  plugins: ['~/plugins/vuetify'],
+   ** Customize the progress-bar color
+   */
+  loading: { color: '#fff' },
+  /*
+   ** Global CSS
+   */
+  css: [],
+  /*
+   * Vuetify custom variables
+   */
+  customVariables: ['~/assets/variables.scss'],
```

Comment:
Oops, I meant top-level Vuetify module option i.e. `vuetify.customVariables` (along `optionsPath`)

Reply:
Sorry for the delay I was taking a long needed vacation lol, please notify me if any more changes are necessary or if there are any more possible improvements to this example!